.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ experimental:
 templates:
   job_template: &job_template
     docker:
-      - image: gcr.io/datadoghq/agent-circleci-runner:v30370116-6cfed421
```

Comment:
Could you use `v31342166-4135b1c3` instead? 

The CI was fixed to ensure the image are also available through public registries, while this tag isn't.
The only other change is related to switching to the old official xz-utils repository instead of github.